### ISSUE TITLE
html.vim doesn't exist.

### ISSUE THREAD

**Issue by HoseynHeydari** [dragfire/Improved-Syntax-Highlighting-Vim#1]
When I open .java files with vim below error appears.
`E484: Can't open file /Users/Hoseyn/.vim/syntax/html.vim`
I commented lines from `153` to `173` of `java.vim` file.
Error doesn't appear again.
Is it ok to remove does lines?

**Comment by dragfire** [comment]
@HoseynHeydari Looks like the error was caused by your `html.vim` not because of `java.vim`.

**Comment by HoseynHeydari** [comment]
@dragfire That's right.  I look at `/Users/Hoseyn/.vim/syntax/` folder again. The `html.vim` wasn't there. I put it there and uncomment lines `153` to `173`. There is no error anymore.

**Comment by HoseynHeydari** [comment]
Where can I find `html.vim` file?
